zilliqa/src/crypto.rs

```diff
@@ -291,6 +290,28 @@ impl Hash {
     }
 }
 
+impl Serialize for Hash {
+    fn serialize<S>(&self, serializer: S) -> std::prelude::v1::Result<S::Ok, S::Error>
+    where
+        S: serde::Serializer,
+    {
+        hex::encode(self.0).serialize(serializer)
```

Comment:
Are we sure we want to serialise hashes as a hex-string rather than a byte array?